tests/rgw/utils.py

```diff
@@ -0,0 +1,22 @@
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def clone_the_repo(config, node, path_to_clone):
+    """
+    clone the repo on to test node
+    :param config: test config
+    :param node: ceph rgw node
+    :param path_to_clone: the path to clone the repo
+
+    """
+    log.info("cloning the repo")
+    branch = config.get("branch", "master")
+    log.info(f"branch: {branch}")
+    repo_url = config.get(
+        "git-url", "https://github.com/red-hat-storage/ceph-qe-scripts.git"
+    )
+    log.info(f"repo_url: {repo_url}")
+    git_clone_cmd = f"sudo git clone {repo_url} -b {branch}"
+    node.exec_command(cmd=f"cd {path_to_clone} ; {git_clone_cmd}")
```

Comment:
Would dynamic directory(`path_to_clone`) be good? and return repository path. Just to avoid any issues due to configuration, If we have parallel tests which uses this method on same node.

Reply:
I added a TODO to take by some one else in next PRs